### DISCUSSION TITLE
Give a name to the main block manager ?

### DISCUSSION THREAD

**Discussion by TomAtTeam** [GrapesJS/grapesjs#3963] [Q&A]
Hi,

I would like to add a name to the block manager on the right so I can organize better my builder.

In this case I would like to give the name "Basic" to the block manager like this :

How can I proceed ?

Thank you.

Thomas

**Comment by ronaldohoch** [comment]
![image](https://user-images.githubusercontent.com/2287371/142624339-e30eff01-15f5-41c2-9b31-d051cf22fd9e.png)
Are you saying this type of organization?

**Reply by TomAtTeam** [reply]
Exactly ! 

I know how to do it for a customed block manager but not for the main one.

**Reply by ronaldohoch** [reply]
In my case, i've cloned and made a custom build of this repo: https://github.com/artf/grapesjs-blocks-basic

But, you can pass the category to plugin options:

```html
<script type="text/javascript">
  var editor = grapesjs.init({
    fromElement: 1,
    container: "#gjs",
    plugins: ["gjs-blocks-basic"],
    pluginsOpts: {
      "gjs-blocks-basic": {
        category:"Custom" // <----- here
      }
    }
  });
</script>
```

As described in this part of readme: https://github.com/artf/grapesjs-blocks-basic#options
![image](https://user-images.githubusercontent.com/2287371/142630300-b5fd23ac-b185-4d88-a3ee-97de96dcf4b4.png)

**Reply by TomAtTeam** [reply]
It still doesn't work and I think it's because I already have a lot of code in my var editor. 

Thank you very much for your reply !

**Reply by ronaldohoch** [reply]
Don't worry, let's solve it :D
Can you provide some code in jsfiddle?
Take this as base: https://jsfiddle.net/szLp8h4n
